ajax/libs/videojs-chromecast/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "videojs-chromecast",
+  "version": "1.1.1",
+  "description": "Displays a Chromecast button in the control bar. The button is only shown if the Google Cast extension is installed and a Chromecast is currently available.",
+  "keywords": [
+    "videjs",
+    "chromecast",
+    "plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kim-company/videojs-chromecast.git"
+  },
+  "author": {
+    "name": "KIM Keep In Mind GmbH, srl",
+    "email": "info@keepinmind.info"
+  },
+  "license": "MIT",
+  "npmName": "videojs-chromecast",
```

Comment:
@pvnr0082t @PeterDaveHello indeed, the repo for this PR is: https://github.com/kim-company/videojs-chromecast.